src/Model/Module.php

```diff
@@ -188,6 +188,19 @@ public function changeBranch($branch)
     }
 
     /**
+     * Gets all tags that exist in the repository
+     *
+     * @return array
+     */
+    public function getTags()
+    {
+        $repo = $this->getRepository();
+        $result = $repo->run('tag');
+        $tags = preg_split('~\R~u', $result);
```

Comment:
\R is linebreaks... it matches \r, \n, and \r\n.

This is what we used in `Convert::nl2os`